azext_edge/edge/params.py

```diff
@@ -619,6 +619,13 @@ def load_iotops_arguments(self, _):
             help="The path to a custom IoT Operations deployment template. Intended for advanced use cases.",
             deprecate_info=context.deprecate(hide=True),
         )
+        context.argument(
+            "dataflow_profile_instances",
```

Comment:
the dataflow profile should have a name right (what is the default then)? a bit weird that only a number is exposed...

Reply:
The name is 'profile' and should not be changed as DOE depends on a static name.

Reply:
oh then I guess applying the dataflow crds isn't valid lol

Reply:
One can create other profiles if they want, we are just ensuring the default one is deployed with init.